src/Tracking.php

```diff
@@ -4,11 +4,95 @@
 
 class Tracking extends ClientBase
 {
-    public function getSummary($trackingPin)
+
+    /**
+     * Get the shipping rates for the given locations and weight.
+     *
+     * @param string $id
+     *   The tracking identification number, either a pin or dnc.
+     * @param string $type
+     *   The tracking identification type, either pin or dnc. Defaults to pin.
+     * @param array $options
+     *   The options to pass along to the Guzzle Client.
+     *
+     * @see https://www.canadapost.ca/cpo/mc/business/productsservices/developers/services/tracking/trackingsummary.jsf
+     * @return \DOMDocument
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getSummary($id, $type = 'pin', array $options = [])
+    {
```

Comment:
Might be good to throw an `InvalidArgumentException` if a non-supported type is given.